package.json

```diff
@@ -80,6 +80,8 @@
     "devDependencies": {
         "@playwright/test": "^1.32.1",
         "@sentry/cli": "^2.2.0",
+        "marked": "4.3.0",
+        "dompurify": "^3.0.2",
```

Comment:
move to deps

Reply:
Already in dependencies, removed from devdep